kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveDriver.java

```diff
@@ -152,6 +160,44 @@ public KeyValueBytesStoreSupplier globalKv(final String name) {
     return new ResponsiveGlobalKeyValueBytesStoreSupplier(client, name, executor);
   }
 
+  public <K, V> StoreBuilder<TimestampedKeyValueStore<K, V>> timestampedKeyValueStoreBuilder(
+      final String name,
+      final Serde<K> keySerde,
+      final Serde<V> valueSerde
+  ) {
+    return new TimestampedKeyValueStoreBuilder<>(
+        timestampedKv(name),
+        keySerde,
+        valueSerde,
+        Time.SYSTEM) {
+      private final Logger log =
+          new LogContext(String.format("store-builder [%s]", name)).logger(ResponsiveDriver.class);
+
+      @Override
+      public StoreBuilder<TimestampedKeyValueStore<K, V>> withLoggingEnabled(
+          final Map<String, String> config
+      ) {
+        final String cleanupPolicy = config.get(TopicConfig.CLEANUP_POLICY_CONFIG);
+
+        if (cleanupPolicy == null || cleanupPolicy.equals(TopicConfig.CLEANUP_POLICY_COMPACT)) {
+          log.debug("Overriding the changelog topic cleanup.policy from compact to delete");
+          final Map<String, String> configCopy = new HashMap<>(config);
+          configCopy.put(TopicConfig.CLEANUP_POLICY_CONFIG, TopicConfig.CLEANUP_POLICY_DELETE);
+          return super.withLoggingEnabled(configCopy);
+
+          // We allow both [delete] and [compact, delete]
```

Comment:
what's the benefit to supporting `[compact, delete]` if we're manually truncating? also does that work (can you truncate a `[comapct, delete]` topic)?

Reply:
> can you truncate a [compact, delete] topic

That's actually a good question -- I misremembered the thing about not being able to truncate compacted topics (actually I thought I'd tested that, it definitely worked locally -- don't suppose you have the exception/error msg on hand or saved anywhere? Would be good to verify the case where both are set)

Reply:
I just tested it with CCloud - it works with `compact,delete` but fails with the following error when it's just `compact`:
```
org.apache.kafka.common.errors.PolicyViolationException: Request parameters do not satisfy the configured policy.
...
Caused by: org.apache.kafka.common.errors.PolicyViolationException: Request parameters do not satisfy the configured policy.
```